regression_logp.py

```diff
@@ -0,0 +1,168 @@
+# Example regression script using neural fingerprints.
+#
+# Compares Morgan fingerprints to neural fingerprints.
+
+import os
+import autograd.numpy as np
+import autograd.numpy.random as npr
+import smile as sm
+from smile import flags, logging
+
+from neuralfingerprint import load_data
+from neuralfingerprint import build_morgan_deep_net
+from neuralfingerprint import build_conv_deep_net
+from neuralfingerprint import normalize_array, adam
+from neuralfingerprint import build_batched_grad
+from neuralfingerprint.util import rmse
+
+from autograd import grad
+
+flags.DEFINE_string("data_path","/home/xiaozhi/datasets/drugs/clean_17p_v1_splited/","cleaned data folder path")
```

Comment:
Please point to a common space or leave blank, if leave blank, please give an example usage of this.